interfaces/cython/cantera/reactor.pyx

```diff
@@ -859,12 +884,17 @@ cdef class ReactorNet:
         self._reactors.append(r)
         self.net.addReactor(deref(r.reactor))
 
-    def advance(self, double t):
+    def advance(self, double t, pybool apply_limit=True):
         """
-        Advance the state of the reactor network in time from the current
-        time to time *t* [s], taking as many integrator timesteps as necessary.
+        Advance the state of the reactor network in time from the current time
+        towards time *t* [s], taking as many integrator timesteps as necessary.
+        If *apply_limit* is true and an advance limit is specified, the reactor 
+        state at the end of the timestep is estimated prior to advancing. If 
+        the difference exceed limits, the step size is reduced by half until 
```

Comment:
Technically, the step size is not reduced (the step size is still controlled by the integrator), right? It should the end time that is reduced by half.

Reply:
Correct.

Reply:
Correct.